### PR TITLE
Add Parseable to powered by page

### PR DESCRIPTION
**Pull request by nitisht**

### CODE REVIEW

File: powered_by.md [hunk 1]
```diff
@@ -154,6 +154,7 @@ short description of your use case.
 * **[pandas][12]:** data analysis toolkit for Python programmers. pandas
   supports reading and writing Parquet files using pyarrow. Several pandas
   core developers are also contributors to Apache Arrow.
+* **[Parseable][51]:** Log analytics platform built for scale and usability. Ingest logs from anywhere and unify logs with Parseable.
```
Comment (kou) [inline powered_by.md:157]: How to use Apache Arrow?

Comment (nitisht) [reply powered_by.md:157]: It uses arrow Recordbatch and flight as intermediary data format. https://github.com/parseablehq/parseable/blob/main/server/src/utils/arrow.rs

Comment (kou) [reply powered_by.md:157]: Thanks. Could you add it too?

Comment (nitisht) [reply powered_by.md]: done!

**Review by kou — APPROVED** [review]
+1

**Pull request merged**